pretix_eth/templates/pretix_eth/pending.html

```diff
@@ -1,12 +1,21 @@
 {% load i18n %}
 {% load static %}
-<a id="pretix-eth-qr-anchor" href="yolo" class="collapse"></a>
+
+<a id="pretix-eth-qr-anchor" href="ethereum:{% blocktrans %}Deposit {{ amount }} {{ coin }} into {{ payment_info }}{% endblocktrans %}" class="collapse"></a>
+<br/><br/>
+<p>
+    <strong>{% blocktrans %}Deposit {{ amount_human }} {{ coin }} into {{ payment_info }}{% endblocktrans %}</strong>
+</p>
+Either via this <a href="https://eips.ethereum.org/EIPS/eip-681">ERC-681</a> QR-Code:<br/><br/>
 <div id="pretix-eth-qr-div"></div>
+<p>
+<script type="text/javascript" src="{% static "pretix_eth/check.js" %}"></script>
 <script type="text/javascript" src="{% static "pretix_eth/generate_qrcode.js" %}"></script>
+<a href="https://checkout.web3connect.com/?currency={{ coin }}&amount={{ amount }}&to={{ payment_info}}&callbackUrl={{ request.build_absolute_uri |urlencode }}">Or Pay via Web3Connect</a>
```

Comment:
Add `target="blank" rel="noreferrer noopener"` to the anchor tag to open the checkout page in a new tab securely

Reply:
good suggestion - thanks!